### ISSUE TITLE
2021-4-26

### ISSUE THREAD

**Issue by ssfc** [ssfc/wangdao#525]
201What is logits[train_mask]? NumTrainingSamples (140) * numClasses (7); (2021-3-30)
202What is pytorch mean? Returns the mean value of all elements in the input tensor. (2021-3-31)
203Download video; 
204船舶检测 | 计算机视觉来看苏伊士运河堵船 https://hub.baai.ac.cn/view/7346 (2021-3-31)
205AI模型性能上不去？这真的不怪我，ImageNet等数据集每100个标签就错3个！ https://hub.baai.ac.cn/view/7339 (2021-3-31)
206【资源推荐】清华大学提出中文预训练语言模型XLNet-tiny https://hub.baai.ac.cn/view/7344 (2021-3-31)
207人人都想做模型工作，而不是数据工作 https://hub.baai.ac.cn/view/7325 (2021-3-31)
208长尾分布 | 基于对比学习的长尾图像分类混合网络（CVPR21） https://hub.baai.ac.cn/view/7343 (2021-3-31)
209What is 注意力机制?  赋予不同的权重; (2021-3-31)
